server/shared/database/seeds/users.ts

```diff
@@ -0,0 +1,15 @@
+import { EntityManager } from '@mikro-orm/postgresql';
+import roles from '../../../user/roles';
+import times from 'lodash/times';
+import User from '../../../user/model';
+
+async function seedUsers(em: EntityManager): Promise<void> {
+  const users = times(10, idx => {
+    const suffix = idx || '';
+    const role = idx ? roles.USER : roles.ADMIN;
+    return new User(`User ${suffix}`, 'Example', `user${suffix}@example.org`, role);
+  });
```

Comment:
This logic seems a bit hard to read. Why not separate admin and user seeding logic - we could have completely different credentials too, i.e. `admin@example.org`?